start.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+set -euo pipefail
+
+result=$(curl -sL ${SRC_URL} | curl -sL -H "Authorization: Bearer ${API_KEY}" --data-binary @- ${DST_URL})
+
+if [ ! -z "$result" ]
```

Comment:
If you just run `curl -sL ${SRC_URL} | curl -sL -H "Authorization: Bearer ${API_KEY}" --data-binary @- ${DST_URL}` directly, the `set -euo pipefail` should fail the script in an error.

Reply:
That only applies to the scenario when curl fails to connect. I'm also checking the http response by saving stdout of the post request. If post succeeds, result will be empty.

Reply:
If post returns 4xx, curl will exit with 0 nevertheless. But our api will return a message json.